drivers/video/fb.c

```diff
@@ -388,6 +388,7 @@ static int fb_ioctl(FAR struct file *filep, int cmd, unsigned long arg)
 
           DEBUGASSERT(fb->vtable != NULL &&
                       fb->vtable->getoverlayinfo != NULL);
+          memset(&oinfo, 0, sizeof(oinfo));
```

Comment:
So the `getoverlayinfo` is not setting all the fields?

Reply:
The driver will return the corresponding overlay information by reading the `overlay` field of `fb_overlayinfo_s`.

Reply:
yes, but why memset-ing to 0 is needed? I see that `overlay` field is not accessed here. Only
```
fb->fbmem = oinfo.fbmem;
fb->fblen = oinfo.fblen;
fb->bpp = oinfo.bpp;
```
the only thing that I can think about is that `getoverlayinfo` will not update those fields, so I'm asking is that really true?

Reply:
overlay/display is an input field, memset ensure it contain a determinate value.

Reply:
@xiaoxiang781216 thank you for explanation.